nginx/etc/nginx/conf.d/api.conf

```diff
@@ -22,6 +24,16 @@ server {
         proxy_pass http://api-server-upstream;
     }
 
+    location ~ ^\/api\/exports\/.*\/files\/.* {
+        limit_req zone=download burst=5;
```

Comment:
I'm not 100% sure what the UX for this will be, but if a user wants to download some number of things within the short period of time, maybe we should add `nodelay` here to cut them a little slack. Otherwise, it seems like the behavior will be to throttle them in a way that seem like any of the requests categorized under `burst` will fail for taking too long.